plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/capability/CapabilityList.js

```diff
@@ -35,9 +36,24 @@ Ext.define('NX.coreui.view.capability.CapabilityList', {
       dataIndex: 'state',
       hideable: false
     },
-    { text: NX.I18n.get('ADMIN_CAPABILITIES_LIST_TYPE_COLUMN'), dataIndex: 'typeName', flex: 1 },
-    { text: NX.I18n.get('ADMIN_CAPABILITIES_LIST_DESCRIPTION_COLUMN'), dataIndex: 'description', flex: 1, groupable: false },
-    { text: NX.I18n.get('ADMIN_CAPABILITIES_LIST_NOTES_COLUMN'), dataIndex: 'notes', flex: 1 }
+    {
+      text: NX.I18n.get('ADMIN_CAPABILITIES_LIST_TYPE_COLUMN'),
+      dataIndex: 'typeName',
+      flex: 1
+    },
+    {
+      text: NX.I18n.get('ADMIN_CAPABILITIES_LIST_DESCRIPTION_COLUMN'),
+      dataIndex: 'description',
+      flex: 1,
+      groupable: false,
+      renderer: NX.ext.grid.column.Renderers.emptyData
```

Comment:
Perhaps 'optionalData' instead, otherwise at first glance it appears that the column is intended to be empty